get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
```

Comment:
thanks @beaucollins. how about I suggest that we either stick with the documented format or update the project README?